3.3999999999999986
1.0
4
-20
2
-20
5
-20
6
-20
244
-20
216391
[4, 5, 5, 2, 44, 66995583]
[4, 3, 7, 447, 4176, 8868642694444]
Error: RuntimeError: Out of bounds memory access (evaluating 'instance.exports._start()')
Stack: python.wasm.wasm-function[_PyMem_SetDefaultAllocator]@[wasm code]
python.wasm.wasm-function[_PyImport_Fini2]@[wasm code]
python.wasm.wasm-function[Py_RunMain]@[wasm code]
python.wasm.wasm-function[pymain_main]@[wasm code]
python.wasm.wasm-function[Py_BytesMain]@[wasm code]
python.wasm.wasm-function[main]@[wasm code]
python.wasm.wasm-function[__main_void]@[wasm code]
python.wasm.wasm-function[_start]@[wasm code]
start@module-ready-wasi-py.js:498:32
@module-ready-wasi-py.js:5225:15
